app/lib/bk/compat/models/steps/circleci/circleci_step.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+module BK
```

Comment:
while it is not failing, I'd suggest adding the requirements for this file explicitly (if you move the file, this will need changing as well)

```suggestion

require_relative '../command'

module BK
```

Reply:
Adjusted - I've moved this over into `steps/` dir so effectively the requirement is effectively `command` 👍